.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedParser.cs

```diff
@@ -118,6 +118,11 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refTime)
             {
                 pr = this.Config.DateTimeAltParser.Parse(er, referenceTime);
             }
+            else if (er.Type.Equals(Constants.SYS_DATETIME_TIMEZONE))
+            {
+                // Commented out for now to not affect the package release.

```

Comment:
Shouldn't this say something like "not affect output" instead of "package release"?